code/handling/SpellController.php

```diff
@@ -197,8 +197,7 @@ public function index() {
 	 */
 	protected function setHeaders() {
 		// Set headers
-		HTTP::set_cache_age(0);
-		HTTP::add_cache_headers($this->response);
+		HTTPCacheControl::singleton()->disableCache();
```

Comment:
This file doesn't exist in versions of SilverStripe lower than 3.7, so we'll need to have a `class_exists` check here and use the previous code if it's in place

Reply:
👍

Reply:
Updated :)